sw/device/silicon_creator/rom/keys/real/rsa/hsmtool/earlgrey_test_signatures.hjson

```diff
@@ -0,0 +1,74 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+
+// The earlgrey test signature were generated with the following command:
+// hsmtool -t <token> -u user -p <pin> exec earlgrey_test_signatures.hjson
+//
+// The input file "test.txt" is simply the file generated by:
+// echo -n "test" > test.txt
+[
+    {
+      "command": "rsa-sign",
+      "id": null,
+      "label": "earlgrey_a0_test_0",
+      "format": "PlainText",
+      "little_endian": false,
```

Comment:
whose endianness does this field refer to and why is it false?

Reply:
this is an unfortunate accident that this shows up in the command file.  `false` is the default value for the `little_endian` setting.

This is meant to be set to `true` when we sign firmware hashes because OT's RV32i is little-endian and `opentitantool image digest <foo>` emits the hash in little-endian order.  When `true`, hsmtool reverses the input hash, signs and reverses the result so the signature can be attached to the manifest without any additional transformation.

For this test, the input isn't a hash - it's plaintext which `hsmtool` then hashes, signs and emits.  The `xxd` pipeline documented in the unit test does the appropriate reversal for this case.